lizmap/modules/lizmap/classes/qgisForm.class.php

```diff
@@ -512,17 +563,40 @@ public function saveToDb($feature = null)
         $dataFields = $this->dbFieldsInfo->dataFields;
         $geometryColumn = $this->dbFieldsInfo->geometryColumn;
 
+        editorForm();
```

Comment:
`$something = $this->editorForm()`, right?

Reply:
```suggestion
```
No just typo